src/mono/sample/wasm/browser-advanced/index.html

```diff
@@ -9,6 +9,10 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="modulepreload" href="./main.js" />
   <link rel="modulepreload" href="./dotnet.js" />
+  <link rel="preload" href="./mono-config.json" as="fetch" crossorigin="anonymous">
```

Comment:
Could we add a rewrite step to the WasmAppBuilder?

Reply:
I'm not sure if adding all resources and DLLs is good idea. If we download all DLLs in parallel with .wasm it would slow down the .wasm, which we need earlier than DLLs.

Reply:
I'm not saying that all resources should be generated as preloads. I'm just against making the list hardcoded in samples.

Reply:
I think that the loader will further evolve in Net8, see https://github.com/dotnet/runtime/issues/70762